index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=inter' rel='stylesheet'>
+    <link rel="stylesheet" href="./style.css">
+    <title>Porfolio</title>
+</head>
+<body>
+    <header>
+        <div class="logo-container">
+            <h2>My Logo</h2>
+        </div>
+        <div class="open_mobile_menu_icon">
+            <img src="Icons/Menu.svg" alt="">
+        </div>
```

Comment:
- [ ] The hamburger button is on the right and should have a nav tag. The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices. :+1: